clients/besu/besu.sh

```diff
@@ -153,6 +153,11 @@ if [ "$HIVE_TERMINAL_TOTAL_DIFFICULTY" != "" ]; then
     RPCFLAGS="$RPCFLAGS --engine-host-allowlist=* --engine-jwt-enabled --engine-jwt-secret /jwtsecret"
 fi
 
+# Enable kzg trusted setup post Cancun
+if [ "$HIVE_CANCUN_TIMESTAMP" != "" ]; then
+    FLAGS="$FLAGS --kzg-trusted-setup=/kzg.txt"
+fi
+
```

Comment:
I wonder, doesn't besu ship with the default trusted setup like all other clients?

Reply:
Not yet at least - we get thrown this error without including it as a flag:
```
WARN  | Besu | Failed to start Besu...
[6efa3097] picocli.CommandLine$ParameterException:
 --kzg-trusted-setup is mandatory when providing a custom genesis that support data blobs
```

I'm assuming it will have the default setup for mainnet so I will dig deeper and ask around!

Reply:
I believe it ships:
```
# Copy KZG-trusted-setup
COPY --from=builder /besu/config/src/main/resources/kzg-trusted-setups/mainnet.txt /kzg-trusted-setup.txt
```

Reply:
It ships, but if you specify your own genesis you need to also provide a corresponding setup if that genesis supports datagas.